src/cache/rax_cache_storage/rax_cache_storage.h

```diff
@@ -6,7 +6,7 @@
 
 #ifndef RAX_CACHE_STORAGE_H
 #define RAX_CACHE_STORAGE_H
-#include "../result_set_cache_includes.h"
+#include "../cache_includes.h"
 #include "../cache_data.h"
 #include "rax/rax.h"
 
```

Comment:
Seems like this entire file is a wrapper over rax, let's use rax directly in cache.h/c and remove this file.

Reply:
done